nifi/data_source_root_process_group.go

```diff
@@ -0,0 +1,32 @@
+package nifi
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceRootProcessGroup() *schema.Resource {
```

Comment:
Does it need to be registered anywhere?

Reply:
```
DataSourcesMap: map[string]*schema.Resource{
		    "nifi_root_process_group":   dataSourceRootProcessGroup(),
        },
```
in provider.go

Reply:
Could you please add it to the PR so the review can be finalized and merged.